Dockerfile

```
@@ -1,11 +1,13 @@
 FROM golang:1.12.4 AS builder
 
-WORKDIR $GOPATH/src/github.com/nats-io/nats-account-server
+WORKDIR /src/nats-account-server
 
-MAINTAINER Waldemar Quevedo <wally@synadia.com>
+LABEL maintainer "Waldemar Quevedo <wally@nats.io>"
+LABEL maintainer "Stephen Asbury <sasbury@nats.io>"
 
 COPY . .
 
+RUN go mod download
 RUN CGO_ENABLED=0 go build -v -a -tags netgo -installsuffix netgo -o /nats-account-server
```

For the dependencies, maybe we could add `GO111MODULE=on` and remove `RUN go mod download` to avoid adding an extra layer to the container.
```suggestion
RUN GO111MODULE=on CGO_ENABLED=0 go build -v -a -tags netgo -installsuffix netgo -o /nats-account-server
```

---

I have had issues with the env var for modules, i find raw folders much safer. How much extra do you think it is?

---

going to merge, lets dig in on this when we have time, i am all for removing layers, but worried because this may require changing what is in the repo (dependency wise)